### PR TITLE
Updates for 0.12

### PR DESCRIPTION
**Pull request by paluh**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -18,12 +18,13 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-prelude": "^3.1.0",
-    "purescript-foldable-traversable": "^3.4.0",
-    "purescript-strings": "^3.3.0"
+    "purescript-prelude": "^4.1.0",
+    "purescript-foldable-traversable": "^4.0.1",
+    "purescript-strings": "^4.0.0",
+    "prelude": "^0.3.7"
```
Comment (LiamGoodacre) [inline bower.json]: This extra `prelude` looks like a mistake?

**Comment by LiamGoodacre** [comment]
Thanks!

**Comment by paluh** [comment]
Yeah, sorry - it was a mistake!

I've overwritten my fork so it should be clean merge now.

**Comment by paluh** [comment]
@LiamGoodacre,

Sorry for bothering you but I like this lib ;-)

Should I fix anything else so you could merge this?

**Comment by LiamGoodacre** [comment]
Wow this was literally a year ago :sweat_smile:
I'll merge and release now!  Thanks for the reminder

**Comment by LiamGoodacre** [comment]
Thanks again!
https://pursuit.purescript.org/packages/purescript-moldy/3.0.0
https://github.com/LiamGoodacre/purescript-moldy/releases/tag/v3.0.0

**Pull request merged**